uobtheatre/bookings/serializers.py

```diff
@@ -164,10 +158,25 @@ def get_price_break_down(self, booking):
         return serialized_price_breakdown.data
 
 
-class CreateTicketSerializer(AppendIdSerializerMixin, serializers.ModelSerializer):
+class CreateTicketSerializer(serializers.ModelSerializer):
```

Comment:
This doesnt really need to be a create serializer any more as it uses the read/write only stuff.
```suggestion
class TicketSerializer(serializers.ModelSerializer):
```